BlazarUI/app/scripts/components/build/Build.jsx

```diff
@@ -12,11 +12,24 @@ import SectionLoader from '../shared/SectionLoader.jsx';
 import Icon from '../shared/Icon.jsx';
 import { BUILD_ICONS } from '../constants';
 import BuildCommits from './BuildCommits.jsx';
+import Button from 'react-bootstrap/lib/Button';
 
 class Build extends Component {
 
-  render() {
+  getAnchorButtons() {
+    return (
+      <div className="btn-group-vertical anchor-container" role="group">
+          <Button className="anchor-button" onClick={() => window.scrollTo(0, 0)} title="Go to top">
```

Comment:
instead of the entire function in the markup, maybe put it in the [helpers file](https://github.com/HubSpot/Blazar/blob/master/BlazarUI/app/scripts/components/Helpers.js) file, that way we can keep the markup a little more slim and it can be reused elsewhere